numba/targets/gdb_hook.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import print_function, absolute_import
```

Comment:
This file needs to be added to .flake8 for pep8 checking.

Reply:
the files listed in .flake8 are those being excluded, not included.

Reply:
On that note, this file is failing several flake8 checks:

https://dev.azure.com/numba/numba/_build/results?buildId=482

Reply:
PEP8 fix applied in 43d1bc06ede5ea4d8c6adae8e382f7edd83b43e7